Generic_AdminActions_Test.php

```diff
@@ -47,25 +47,52 @@ function w3tc_test_redis() {
 		else {
 			$success = true;
 
-			foreach ( $servers as $server ) {
-				@$cache = Cache::instance( 'redis', array(
-						'servers' => $server,
-						'persistent' => false,
-						'password' => $password,
-						'dbid' => $dbid
-					) );
+			if ( W3TC_REDIS_CLUSTER === true ) {
```

Comment:
I better check would be:
`if ( defined( 'W3TC_REDIS_CLUSTER' ) && W3TC_REDIS_CLUSTER ) {`